contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -30,6 +29,10 @@ abstract contract VaultHub is PausableUntilWithRoles {
         mapping(address => uint256) vaultIndex;
         /// @notice allowed beacon addresses
         mapping(bytes32 => bool) vaultProxyCodehash;
+        /// @notice maximum number of vaults that can be connected to the hub
```

Comment:
simultaneously or ever connected?

Reply:
Ever. However, this is subject to change based on the operator grid, I believe.

Reply:
Fixed in c483544